### PR TITLE
ISPN-15144 Protostream 4.6.5

### PR DESCRIPTION
**Pull request by wburns**
* Change test back to a lower size estimate|

https://issues.redhat.com/browse/ISPN-15144

### CODE REVIEW

File: core/src/test/java/org/infinispan/stats/SingleStatsTest.java [hunk 1]
```diff
@@ -87,8 +87,8 @@ protected void configure(ConfigurationBuilder cfg) {
          memoryConfigurationBuilder.maxCount(size);
       } else {
          if (storageType == StorageType.OFF_HEAP) {
-            // When expiration eviction is used it stores the internal metadata which we will estimate as 24 bytes
-            long amountPerEntry = OFF_HEAP_SIZE + (evictionStrategy.isExceptionBased() ? 24 : 0);
+            // When expiration eviction is used it stores the internal metadata which we will estimate as 12 bytes
+            long amountPerEntry = OFF_HEAP_SIZE + (evictionStrategy.isExceptionBased() ? 12 : 0);
```
Comment (pruivo) [inline core/src/test/java/org/infinispan/stats/SingleStatsTest.java:91]: previously, the `amountPerEntry` was always `OFF_HEAP_SIZE`.

Comment (wburns) [reply core/src/test/java/org/infinispan/stats/SingleStatsTest.java:91]: I am aware, and that was not correct.

Comment (wburns) [reply core/src/test/java/org/infinispan/stats/SingleStatsTest.java:91]: We can probably lower OFF_HEAP_SIZE, but the exception based size needs to be accounted for.

Comment (wburns) [reply core/src/test/java/org/infinispan/stats/SingleStatsTest.java]: I have updated it with exact memory allocation sizes now.

**Comment by ryanemerson** [comment]
Thanks @wburns

**Pull request merged**